DotNet/Report/Jobs/GenerateDataAcquisitionRequestsForPatientsToQuery.cs

```diff
@@ -56,15 +57,12 @@ public async Task Execute(IJobExecutionContext context)
                 _logger.LogInformation(
                     $"Executing GenerateDataAcquisitionRequestsForPatientsToQuery for MeasureReportScheduleModel {schedule.Id}");
 
-                if ((schedule.PatientsToQuery?.Count ?? 0) > 0)
-                {
-                    ProducerConfig config = new ProducerConfig()
-                    {
-                        ClientId = "Report_DataAcquisitionScheduled"
-                    };
+                var submissionEntries = await _database.SubmissionEntryRepository.FindAsync(e => e.ReportScheduleId == schedule.Id);
+                var itemsToEvaluate = submissionEntries.Where(x => x.Status == PatientSubmissionStatus.NotEvaluated).Select(x => x.PatientId).ToList();
 
-
-                    foreach (string patientId in schedule.PatientsToQuery ?? new List<string>())
+                if (itemsToEvaluate.Any())
+                {
+                    foreach (string patientId in itemsToEvaluate)
```

Comment:
Can this potentially create  multiple DA requests for same patient?

Reply:
Yes, good catch. I have put a distinct() on the patientid list.